renku/api/datasets.py

```diff
@@ -145,29 +145,36 @@ def with_dataset(self, name=None):
         dataset.to_yaml()
 
     def add_data_to_dataset(
-        self, dataset, url, git=False, force=False, **kwargs
+        self, dataset, urls, git=False, force=False, **kwargs
     ):
         """Import the data into the data directory."""
         dataset_path = self.path / self.datadir / dataset.name
-        git = git or check_for_git_repo(url)
 
-        target = kwargs.pop('target', None)
+        files = []
 
-        if git:
-            if isinstance(target, (str, NoneType)):
-                files = self._add_from_git(
-                    dataset, dataset_path, url, target, **kwargs
-                )
-            else:
-                files = []
-                for t in target:
```

Comment:
`target` can have multiple elements - but this should probably be handled in the `_add_from_*` methods anyway

Reply:
I didn't change this part. github just has issues with following the changes, the diff is a bit of a mess (I indented the whole block and added `for url in urls:` at the top). I.e. this "removed" part is "added" again further down.

Reply:
ah yep I see it, thanks!